test_scripts/test_setups.py

```diff
@@ -100,140 +99,237 @@ class TestPhase(Enum):
+    'gemini_gmos': ['GS_HAM_R400_700', 'GS_HAM_R400_860',
+                    'GN_HAM_R400_885', 'GN_HAM_NS_B600_620',
+                    'GS_HAM_MULTI_R400_700', 'GN_E2V_MULTI_R400_600'],
+    'gemini_flamingos2': ['HK_HK', 'JH_JH'],
+    'gtc_osiris': ['R1000B', 'R1000BMOS', 'R1000RMOS', 'R2500R', 'R2500V'],
+    'keck_deimos': ['600ZD_M_6500', '600ZD_tilted', '1200G_M_7750', '830G_LVM_8400', '830G_M_8100_26',
+                    '830G_M_8500', '830G_L_8100', '1200B_M_5200', '1200G_M_5500', 
+                    '900ZD_M_6000', '1200B_LVM_5200', '900ZD_LVM_5500', 
+                    '830G_M_9000_dither'],
+    'keck_kcwi': ['bh2_4200', 'bl'],
+    'keck_nires': ['NIRES', 'ERIS'],
+    'keck_nirspec': ['LOW_NIRSPEC-1'],
+    'keck_mosfire': ['Y_long', 'J_multi', 'K_long', 'Y_multi', 'long2pos1_H', 'longslit_3x0.7_H', 'mask1_K_with_continuum', 'mask1_J_with_continuum', 'J2_long'],
+    'keck_lris_blue': ['multi_600_4000_d560', 'long_400_3400_d560', 'long_600_4000_d560',
+                      'multi_300_5000_d680', 'multi_600_4000_slitmask'],
+    'keck_lris_blue_orig': ['long_600_4000_d500'],
+    'keck_lris_red': ['long_600_7500_d560', 'multi_1200_9000_d680_1x2',
+                      'multi_600_5000_d560', 'multi_1200_9000_d680',
+                      'multi_400_8500_d560', 'long_600_10000_d680',
+                      'long_400_8500_longread'],  # Longslit read-out mode
+    'keck_lris_red_orig': ['long_300_5000'],
+    'keck_lris_red_mark4': ['long_400_8500_d560', 'long_600_10000_d680'],
+    'lbt_luci': ['LUCI-I', 'LUCI-II'],
+    'lbt_mods': ['MODS1R_Longslit', 'MODS2R_Longslit'],
+    'ldt_deveny': ['DV1', 'DV2', 'DV5', 'DV6', 'DV8', 'DV9'],
+    'magellan_mage': ['1x1'],
+    'magellan_fire': ['FIRE_Echelle', 'FIRE_Long'],
+    'mdm_osmos': ['MDM4K'],
+    'mmt_binospec': ['Longslit_G600', 'Multislit_G270'],
+    'mmt_mmirs': ['HK_zJ', 'J_zJ', 'K_K'],
+    'mmt_bluechannel': ['300l'],
+    'ntt_efosc2': ['gr5', 'gr6'],
+    'not_alfosc': ['grism4', 'grism19'],
+    'p200_dbsp_blue': ['600_4000_d55', '600_4000_d68', '1200_5000_d68'],
+    'p200_dbsp_red': ['316_7500_d55', '600_10000_d55', '1200_7100_d68'],
+    'p200_tspec': ['TSPEC'],
+    'shane_kast_blue': ['452_3306_d57', '600_4310_d55', '830_3460_d46'],
+    'shane_kast_red': ['300_7500_Ne', '600_7500_d55_ret', '600_7500_d57', '600_5000_d46', '1200_5000_d57'],
+    'soar_goodman_red': ['M1','M2'],
+    'tng_dolores': ['LRB'],
+    'vlt_fors2': ['300I', '600Z'],
+    'vlt_sinfoni': ['K_0.8'],
+    'vlt_xshooter': ['VIS_1x1', 'VIS_2x1', 'VIS_2x2', 'VIS_manual', 'NIR', 'UVB_1x1'],
+}
+
+# Tests for full reductions
+_reduce_setups = {}
+for instr in all_setups:
+    _reduce_setups[instr] = {}
+    for setup in all_setups[instr]:
+        _reduce_setups[instr][setup] = [{}]
+
+#all_setups = copy.deepcopy(reduce_setups)
+#all_setups['keck_deimos'].append('QL')
+
+# If you add a test here and it is in QL
+#   you will need may modify the QuickLookTest class 
```

Comment:
I think you can avoid having to modify the QuickLookTest class, see my note in pypeit_tests.py

Reply:
removed